Library/Homebrew/cmd/outdated.rb

```diff
@@ -177,6 +185,14 @@ def outdated_casks
         end
       end
 
+      def disabled_casks
+        if args.named.present?
+          select_disabled(args.named.to_casks)
+        else
+          select_disabled(Cask::Caskroom.casks)
```

Comment:
Can you use `hyperfine` to see how much slower this makes the default `brew outdated`?